app/i18n/oc/sub.php

```diff
@@ -73,51 +73,51 @@
 		'information' => 'Informacions',
 		'keep_min' => 'Nombre minimum d’articles de servar',
 		'kind' => array(
-			'_' => 'Type of feed source',	// TODO
+			'_' => 'Tipe de font de flux',
 			'html_xpath' => array(
-				'_' => 'HTML + XPath (Web scraping)',	// TODO
+				'_' => 'HTML + XPath (Web scraping)',
```

Comment:
Is this already translated?
If yes, please add ` // IGNORE` at the end, so that our pipeline understand, that it is not a translation issue

Reply:
Oh yes, it's best to use the English term here. On my way to change this.